textdb/textdb-dataflow/pom.xml

```diff
@@ -41,6 +41,11 @@
             <artifactId>textdb-storage</artifactId>
             <version>${textdb.version}</version>
         </dependency>
+        <dependency>
+            <groupId>org.mockito</groupId>
```

Comment:
@chenlica No, we didn't use it before. Actually, Mockito is the ideal way of doing unit testing. In this case, it is easier also, because it is better to mock the `IOperator` Interface instead of constructing one. Same applies to `ITuple` and `ISink`